### ISSUE TITLE
[Package Issue]: Microsoft.PowerBI 2.114.864.0 hash mismatch

### ISSUE THREAD

**Issue by butwhoscounting** [microsoft/winget-pkgs#99666]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

Microsoft.PowerBI 2.114.864.0 installer hash does not match.

The InstallerSha256 value found in the package manifest file ([link](https://github.com/microsoft/winget-pkgs/blob/master/manifests/m/Microsoft/PowerBI/2.114.864.0/Microsoft.PowerBI.installer.yaml)) is "CD5EEC6A41E25002D4BDB137996EC35C0267F1D7B633BEFC8E63A6829FF841CE". 
Calculating the SHA256 hash of the PowerBI 2.114.864.0 installer locally on my machine I see that it is instead "ca19538b072c4f8b4de7a7e58b6314eb0676da99edb6548ae0321708e3a33724".

### Steps to reproduce

```winget install Microsoft.PowerBI```

### Actual behavior

```
PS C:\Users\butwhoscounting> winget install Microsoft.PowerBI
Found Microsoft PowerBI Desktop [Microsoft.PowerBI] Version 2.114.864.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.microsoft.com/download/8/8/0/880BCA75-79DD-466A-927D-1ABF1F5454B0/PBIDesktopSetup_x64.exe
  ██████████████████████████████   657 MB /  657 MB
Installer hash does not match.
```

### Expected behavior

The application installation should complete without any hash mismatch related errors.

### Environment

```raw
Windows Package Manager v1.4.10173
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.1413
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0
```

**Comment by BrandonWanHuanSheng** [comment]
The issue is continuing on another link. If you saw wingetbot. Most of the times, look at the https://github.com/microsoft/winget-pkgs/pulls?q=wingetbot This bot is automatic updates on it. I think it is the winget-bot

**Comment by mdanish-kh** [comment]
Resolved in #99731
